sdk/storage/storage-blob/src/utils/utils.node.ts

```diff
@@ -145,33 +145,19 @@ export async function readStreamToLocalFile(
       ws.on("unpipe", () => console.log("ws.unpipe"));
```

Comment:
Why `console.log` is in production code?

Reply:
This was added in `test\utils` in the https://github.com/Azure/azure-sdk-for-js/pull/1783 to console log all the events which would be helpful to debug the events.
And it only runs when `STREAM_DEBUG` env var is present. 
 > // Set STREAM_DEBUG env var to log stream events while running tests

The whole function [`readStreamToLocalFile`](https://github.com/Azure/azure-sdk-for-js/pull/3818/files#diff-7a96f28c33d102f0ead4cf6a1f7cf984R377-R380) has been moved from `test\utils` to `src\utils` here -  #3818.
And moved around once more in the `src\utils` folder here - https://github.com/Azure/azure-sdk-for-js/pull/5161

Reply:
And you're right, it shouldn't have been here in the first place since it's production code, I can log an issue to remove it from here.

Reply:
I changed to have two copy of similar functions. One with console log and error tracking for testing, the other without console log/error tracking for production.